docs/admin/node.md

```diff
@@ -191,6 +191,14 @@ For self-registration, the kubelet is started with the following options:
 Currently, any kubelet is authorized to create/modify any node resource, but in practice it only creates/modifies
 its own.  (In the future, we plan to limit authorization to only allow a kubelet to modify its own Node resource.)
 
+If your cluster runs out short on resources you can easily add more machines to it if your cluster is running in Node self-registartion mode. If you're using GCE or GKE it's done by resizing Instance Group managing your Nodes. It can be accomplished by modifying number of instances on `Compute > Compute Engine > Instance groups > your group > Edit group` [Google Cloud Console page](https://console.developers.google.com) or using gcloud CLI:
+```
+gcloud preview managed-instance-groups --zone compute-zone resize my-cluster-minon-group --new-size 42
+```
+Instance Group will take care of putting appropriate image on new machines and start them, while Kubelet will register its Node with API server to make it available for scheduling. If you scale the instance group down, system will randomly choose Nodes to kill, so if you want to keep everything running all the time you probably shouldn't downsize the cluster twice.
```

Comment:
remove "so if you want to keep everything running all the time you probably shouldn't downsize the cluster twice."